.github/workflows/build_cmake.yml

```diff
@@ -380,15 +427,15 @@ jobs:
       with:
         name: "Html documentation artifacts"
         path: build/html/
-      if: matrix.config.name == 'Ubuntu Latest GCC Release'
+      if: matrix.config.name == 'Ubuntu Latest GCC Release (Intel)' || matrix.config.name == 'Ubuntu Latest GCC Release (Arm)'
```

Comment:
- The HTML and Latex documentation artifacts should only be done for one platform, so for `Ubuntu Latest GCC Release (Intel)` only
- The generation of the internal documentation should only be done for `Ubuntu Latest GCC Release (Intel)`